specification/cognitiveservices/data-plane/TranslatorText/v3.0/readme.md

```diff
@@ -0,0 +1,120 @@
+# Cognitive Services Translator Text SDK
+
+> see https://aka.ms/autorest
+
+Configuration for generating Translator Text SDK.
+
+The current release is `release_3_0`.
+
+``` yaml
+
+tag: release_3_0
```

Comment:
It's okay to have a README in this directory, but it should contain no AutoRest configuration. Please delete this file, or completely replace its contents.